pkg/image/api/helper.go

```diff
@@ -184,7 +184,41 @@ func ParseDockerImageReference(spec string) (DockerImageReference, error) {
 		ref.ID = id
 		break
 	default:
-		// TODO: this is no longer true with V2
+		// Handle multiple segments in form: registry/namespace/namesegment1/namesegment2/.../name
+		// but not this form: http://registry/namespace/name. We don't support using the schema in
+		// the DockerImageReference.
+		if len(repoParts) > 3 && !strings.HasSuffix(repoParts[0], ":") {
```

Comment:
@ncdc We have a tests that verify this ban:

https://github.com/openshift/origin/blob/master/pkg/image/api/helper_test.go#L290-L291

https://github.com/openshift/origin/blob/master/pkg/image/api/helper_test.go#L298-L299